pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -251,13 +265,19 @@ let
 
     inherit meta;
   };
+  platformTag =
+    if stdenv.targetPlatform.isLinux then
+      "manylinux2010_${stdenv.targetPlatform.linuxArch}"
+    else if stdenv.targetPlatform.isDarwin then
+      "macosx_10_9_${stdenv.targetPlatform.linuxArch}"
```

Comment:
Why `macosx_10_9`? It looks like the pypi wheels are using the `macosx_11_0` tag already: https://pypi.org/project/jaxlib/#files.

Reply:
note to self: we should also update to `manylinux_2014` as it seems that's what upstream is doing nowadays as well

Reply:
That comes from https://github.com/google/jax/blob/jaxlib-v0.3.0/build/build_wheel.py#L261-L262, looks like `macosx_11_0` is only used for arm on `0.3.0`.

Reply:
ah I see... very good catch. And I guess for the time being we will only support x86_64-darwin anyhow?